File: PDOO-Napakalaki/Napakalaki/NapakalakiJava/src/NapakalakiGame/DeathBadConsequence.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

public class DeathBadConsequence extends NumericBadConsequence {
    public DeathBadConsequence(String text) {
        super(text,21,MAXTREASURES,MAXTREASURES);
    }

    @Override
    public boolean isDead() {
        return true;
    }
    
    @Override 
    public String toString(){
        return "Estas muerto.";
    }
    
    

    
}

File: PDOO-Napakalaki/Napakalaki/NapakalakiRuby/lib/death_bad_consequence.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require_relative "numeric_bad_consequence.rb"

class DeathBadConsequence < NumericBadConsequence
  
  attr_accessor :death
  
  def initialize(aText, death)
    super(aText, 10, 10, 10)
    @death = death
  end
end
